basiclti/basiclti-portlet/src/java/org/sakaiproject/portlets/IMSBLTIPortlet.java

```diff
@@ -238,9 +242,12 @@ public void doView(RenderRequest request, RenderResponse response)
 						text.append(windowOpen+"\n");
 						text.append("</script>\n");
 					}
-					text.append(rb.getString("new.page.launch"));
-					text.append("<br><a href=\""+iframeUrl+"\" onclick=\""+windowOpen+"\" target=\"BasicLTI\">"+rb.getString("noiframe.press.here")+"</a>");
+					String siteName = ServerConfigurationService.getString(SITE_NAME, SAKAI);
+					title = title!=null ? title : rb.getString("tool.name", "your tool");
```

Comment:
I think title needs to be escaped as it's put into a HTML attribute and should escape any quotes.